app.js

```diff
@@ -29,71 +28,151 @@ li.forEach((i) => {
-//     source: 'https://github.com/fed1k/Personal-Portfolio.github.io',
-//   },
-//   card5 = {
-//     name: 'Fifth Project',
-//     description: 'Lorem ipsum dolor sit ametabore et dolore magna aliqua. Ut enim ad',
-//     technologies: ['HTML/CSS', 'Ruby on Rails', 'Javascript'],
-//     live: 'https://fed1k.github.io/Personal-Portfolio.github.io/',
-//     source: 'https://github.com/fed1k/Personal-Portfolio.github.io',
-//   },
-//   card6 = {
-//     name: 'Sixth Project',
-//     description: 'Lorem ipsum dolor sit amet, conseabore et dolorei',
-//     technologies: ['HTML/CSS', 'Ruby on Rails', 'Javascript'],
-//     live: 'https://fed1k.github.io/Personal-Portfolio.github.io/',
-//     source: 'https://github.com/fed1k/Personal-Portfolio.github.io',
-//   },
-// ];
-
-// const seebtn = document.querySelectorAll('.see');
-// seebtn.forEach((own) => {
-//   own.addEventListener('click', () => {
-//     const conteyner = document.createElement('section');
-//     const body = document.querySelector('body');
-//     conteyner.className = 'mobile';
-//     body.appendChild(conteyner);
-//     conteyner.insertBefore(ul);
-//   });
-// });
\ No newline at end of file
+  return ul;
+}
+
+function image() {
+  const image = document.createElement('img');
+  image.className = 'projectImage';
+  image.src = 'images/Rectangle-21-5.png';
+
+  return image;
+}
+
+function description() {
+  const description = document.createElement('p');
+  description.className = 'projectDescription';
+  description.innerHTML = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.';
+
+  return description;
+}
+
+function button() {
+  const arrName = ['See live', 'Live Source'];
+  const arrIcon = ['fas fa-external-link-alt btnIcon', 'fab fa-github btnIcon'];
+
+  const buttonDiv = document.createElement('div');
+  buttonDiv.className = 'parentButtonDiv';
+
+  for (let i = 0; i < 2; i += 1) {
+    const icon = document.createElement('i');
+    const button = document.createElement('button');
+
+    button.className = 'projectButton';
+    icon.className = arrIcon[i];
+    button.append(arrName[i], icon);
+    buttonDiv.appendChild(button);
+  }
+
+  return buttonDiv;
+}
+
+function popupModel() {
+  heading();
+  languages();
+  image();
+  description();
+  button();
+
+  const div = document.createElement('div');
+  div.className = 'divider';
+
+  mainDiv.className = 'main-popup-div';
+  mainDiv.append(heading(), languages(), image(), description(), button());
+  div.appendChild(mainDiv);
+  const card = document.querySelector('.card1');
+
+  card.appendChild(div);
+}
```

Comment:
- [ ] For the popup the design should be responsive on both the desktop and mobile versions. please look at your mobile version and make it responsive. ensuring to keep the desktop version responsive as well

### Your mobile version currently 
![image](https://user-images.githubusercontent.com/18730106/152664876-7d4681f1-5a2b-4126-8859-927f700d7900.png)